ZitiDesktopEdge.Client/DataStructures/DataStructures.cs

```diff
@@ -8,7 +8,6 @@
 /// </summary>
 namespace ZitiDesktopEdge.DataStructures {
     public enum LogLevelEnum {
-        FATAL = 0,

```

Comment:
i might not remember but i thought these were straight from the c sdk? i don't understand the desire to remove this? why bother? why not just remove it from the UI